walsh_ao6c.js

```diff
@@ -0,0 +1,63 @@
+/* Tyler Walsh
+walsh_ao6c.js
+19WI_INFO_2124_WW
+Thoendel
+1/25/2019 */
+
+//winning numbers: 22, 42, 43, 47, 61;
+
+let lottoPicks = [];
+let powerBallPick = getLottoNumber(); 
+let winnings = 0; 
+
+/* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+
+while(lottoPicks.length < 5) { //If the lottoPicks array has less than 5 items in it, do this then check how many items it has inside.
+
+    let randomNumber = getLottoNumber(); //Generate a random number with this method. These are the lotto numbers.
+
+    lottoPicks.unshift(randomNumber); //Add that random number to the lottoPicks array.
+}
+
+for (var i = 0; i < lottoPicks.length; i++) { //For each item in the lottoPicks array, do this.
+    switch(lottoPicks[i]) { //A switch statement is an easy way to basically make multiple similar "if, else" statements without cluttering the code too much.
+
+        case 22:  //If the number matches any of the winning lottery numbers, add $10 to the prize pool.
+            winnings = winnings + 10; 
+        break; //The break statement means the expression checks for the next case. This means, if you somehow rolled a 22 and a 42 on the same number, you would get $20.

```

Comment:
actually - break breaks you out of the control structure - the switch in this case.
so - the next case (42) will not get checked. The idea of a switch is that you will match ONE case. continue is the keyword that will allow you to continue inside the control structure.
Let me know if you have questions.